server/host_controls.go

```diff
@@ -56,3 +56,41 @@ func (p *Plugin) changeHost(requesterID, channelID, newHostID string) error {
 
 	return nil
 }
+
+func (p *Plugin) muteSession(requesterID, channelID, sessionID string) error {
+	state, err := p.getCallState(channelID, false)
+	if err != nil {
+		return err
+	}
+
+	if state == nil {
+		return errors.New("no call ongoing")
+	}
+
+	if requesterID != state.Call.GetHostID() {
+		if isAdmin := p.API.HasPermissionTo(requesterID, model.PermissionManageSystem); !isAdmin {
+			return errors.New("no permissions to mute session")
+		}
+	}
+
+	ust, ok := state.sessions[sessionID]
+
+	if !ok {
+		return errors.New("session is not in the call")
+	}
+
+	if p.isBot(ust.UserID) {
+		return errors.New("cannot mute the bot")
+	}
```

Comment:
Is this reflected in the client side interface as well?

Reply:
I don't think it needs to be -- we don't see the bots in the call, do we?

Reply:
Right, so why the check? :p They are unable to speak at the moment anyway ehe

Reply:
This was one of those "I may as well, or else I'll get called on it" defensive coding things. :P I'll take it out, np.